--- src/main.cpp.orig	Fri Apr 30 17:16:56 2004
+++ src/main.cpp	Tue Sep 28 20:17:26 2004
@@ -89,10 +89,10 @@
 	// The variable CONFIG_DATA_DIR must have been set by the configure script
 	// Its value is the dataRoot directory, ie the one containing data/ and textures/
 
+    FILE * tempFile = NULL;
 #if !defined(MACOSX) || ( defined(MACOSX) && !defined(XCODE) )
 	// Check the presence of a file in possible data directories and set the
 	// dataRoot string if the directory was found.
-    FILE * tempFile = NULL;
 	tempFile = fopen((string(CONFIG_DATA_DIR) + "/data/hipparcos.fab").c_str(),"r");
 	if (tempFile)
 	{
