--- stellarium-0.5.2/src/stelconfig.cpp.orig	2004-01-21 22:09:00.000000000 +0100
+++ stellarium-0.5.2/src/stelconfig.cpp	2004-01-21 22:49:56.000000000 +0100
@@ -126,8 +126,9 @@
 
 
 #if defined(WIN32) || defined(CYGWIN) || defined(__MINGW32__) || defined(MACOSX)
-	strcpy(global.ConfigDir,dataRoot);
-	strcat(global.ConfigDir,"/config/");
+//	strcpy(global.ConfigDir,dataRoot);
+//	strcat(global.ConfigDir,"/config/");
+    	strcpy(global.ConfigDir,"/etc/X11/stellarium/");
 #else
     char * homeDir = getenv("HOME");
     char tempName2[255];
@@ -187,10 +188,14 @@
 			}
 		}
 		// First launch for that user : set default options
-    	strcpy(tempName,dataRoot);
-    	strcat(tempName,"/config/default_config.txt");
-    	strcpy(tempName2,dataRoot);
-    	strcat(tempName2,"/config/default_location.txt");
+//   	strcpy(tempName,dataRoot);
+	strcpy(tempName,"/etc/X11/stellarium");
+//   	strcat(tempName,"/config/default_config.txt");
+	strcat(tempName,"/default_config.txt");
+//   	strcpy(tempName2,dataRoot);
+	strcpy(tempName2,"/etc/X11/stellarium");
+//   	strcat(tempName2,"/config/default_location.txt");
+	strcat(tempName2,"/default_location.txt");
 		loadConfig(tempName,tempName2);
 		printf("Init config and location files\n");
 		dumpConfig();
